.github/workflows/merge-release-notes.yaml

```diff
@@ -0,0 +1,61 @@
+name: Merge release notes
+
+on:
+  workflow_dispatch:
+
+jobs:
+  merge-release-notes:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set version
+        id: version
+        run: |
+          VERSION=$(echo ${{ github.ref }} | sed -e "s#refs/tags/v##g")
```

Comment:
It looks like we need to trigger a `workflow_dispatch` with specifying a tag. What happens if we specify a branch instead? Maybe an error occurs? Ideally, we should probably add a check for that, but If it doesn't cause any unexpected issues, I believe that should be acceptable.

Reply:
I see. Let me check.

Reply:
@brfrn169 

I checked that invoking this workflow with a branch.
It will cause an error when downloading the ScalaDB's release notes (`Checkout the ScalarDB's release notes` step) and the remaining steps were skipped.

I don't think there are any negative effects if you run it on a branch instead of a tag.

(Of course, I'll consider adding a validation to the Set version job in the future.)

Is it OK to you?

Reply:
Thank you for confirming!